### PR TITLE
[refactor 2] Create dumb Widget, Page, HelpCircle component

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

This PR refactor the pages layout (`Page` component), and what it was called before `AppBody`.

Additionally, creates a dumb component `HelpCircle` that was in the `Page` component. Now it's in the `shared` module (it was used as a shared component accross the site) 

## AppBody
I extracted what we used to call AppBody, made it into a dumb component, created a cosmos Fixture, and reanamed to `Widget`

This is the main Widget used for the Swap and some pages.

image

image

## Page
The page component is the main component used to create pages.

Created a dumb component and some fixture too
Screenshot at Sept 29 14-45-15

## HelpCircle
It was used across the page as a shared component. Now its in `modules/shared/dumb` and has a fixture. 

Note: I didn't fix the fact that the size in the prop is used, but it doesn't scale the SVG. This PR is just to refactor, and I don't want to break any other part of the app which uses this (i see the component size grows, its just the SVG don't scale with it, so its possible that was done on porpouse --> i think its weird)

image

### CODE REVIEW

**Comment by elena-zh** [comment]
Seems that the build failed :(

File: src/cow-react/modules/application/dumb/Widget/index.tsx [hunk 1]
```diff
@@ -1,23 +1,32 @@
 import styled from 'styled-components/macro'
 import { BodyWrapper as BodyWrapperMod } from '@src/pages/AppBody'
-import { transparentize } from 'polished'
 
-export const BodyWrapper = styled(BodyWrapperMod)`
+const Wrapper = styled(BodyWrapperMod)`
```
Comment (alfetopito) [inline src/cow-react/modules/application/dumb/Widget/index.tsx:4]: I thought you said no styles in dumb components?
Refactor still to come or I got the rule wrong?

File: src/cow-react/pages/Account/LockedGnoVesting/index.tsx [hunk 2]
```diff
@@ -32,6 +31,7 @@ import { CurrencyAmount, Currency } from '@uniswap/sdk-core'
 import { getProviderErrorMessage, isRejectRequestProviderError } from 'utils/misc'
 import { claimAnalytics } from 'utils/analytics'
 import { ButtonSize } from 'theme'
+import { HelpCircle } from '@src/cow-react/modules/shared/dumb/HelpCircle'
```
Comment (alfetopito) [inline src/cow-react/pages/Account/LockedGnoVesting/index.tsx:34]:
```suggestion
import { HelpCircle } from 'cow-react/modules/shared/dumb/HelpCircle'
```

**Review by alfetopito — APPROVED** [review]

File: src/cow-react/pages/Account/LockedGnoVesting/index.tsx (hunk 2 above)
Comment (anxolin) [reply src/cow-react/pages/Account/LockedGnoVesting/index.tsx]: addressed in another PR. I only saw later

**Pull request merged**